app/screens/MapScreen.js

```diff
@@ -14,7 +13,14 @@ class MapScreen extends React.Component {
 
 	constructor(props) {
 		super(props);
-		this.state = { label: props.initialInput, x: props.x, y: props.y };
+
+		this.state = {
+			label: '',
+			x: 0,
+			y: 0
+		};
+
+		this.onClick = this.onClick.bind(this);
```

Comment:
Why not just arrow function `onClick`?

Reply:
Isn't it worse for runtime?  But yea, it would be less wordy if we used an arrow.  I guess it doesn't matter either way, so we can leave it as it is unless you have any more thoughts on this?

Reply:
Yeah I'm fine with leaving it.